compiler-plugin-test/src/test/resources/kotlin/IT_EXT_ALGN_for_exampleAggregate.kt

```diff
@@ -0,0 +1,15 @@
+import it.unibo.collektive.aggregate.api.Aggregate
+
+fun Aggregate<Int>.exampleAggregate() {}
+
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
+// END OF BOILERPLATE CODE
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
+
+fun Aggregate<Int>.entry() {
+    alignedOn(0) {
+        for(i in 1..3) {
+            exampleAggregate()
+        }
+    }
```

Comment:
I'd like to have also the case in which we don't call `alignedOn`

Reply:
I'm sorry, do you mean like here or am I missing something?
https://github.com/Collektive/collektive/blob/27365a202b25f1e42c3cb35a61a141c7a270687d/compiler-plugin-test/src/test/resources/kotlin/SIMPLE_IT_for_exampleAggregate.kt#L9-L13

Reply:
You are missing nothing, closing :)